torch_xla/experimental/spmd_fully_sharded_data_parallel.py

```diff
@@ -24,6 +24,8 @@ def _prepare_spmd_partition_spec(param):
   # TODO: should we shard on the maximal dim for param? Then we need
   # another helper for the output.
   partition_spec[0] = "fsdp"
+  if extra_data_axis:
+    partition_spec[0] = ("fsdp", extra_data_axis)
```

Comment:
We usually have this reversed for DCN - `(extra_data_axis, 'fsdp')`. The axes should be in order of increasing network intensity in the mesh, and the order in the partition spec will impact the sharding.